pom.xml

```diff
@@ -49,16 +54,15 @@
         </plugins>
     </build>
 
+    <!-- Thanks for using https://jar-download.com -->
     <modules>
         <module>jdi-light</module>
         <module>jdi-light-examples</module>
         <module>jdi-light-html-tests</module>
         <module>jdi-light-html</module>
-        <module>jdi-light-bootstrap</module>
-        <module>jdi-light-bootstrap-tests</module>
-        <module>jdi-light-bdd</module>
-        <module>jdi-light-bdd-tests</module>
-        <module>jdi-light-bdd-no-po-tests</module>
+        <module>jdi-bdd</module>
+        <module>jdi-bdd-tests</module>
+        <module>jdi-bdd-no-po-tests</module>
```

Comment:
Why are you editing pom file?

Reply:
import sun.reflect.generics.reflectiveObjects.NotImplementedException;
Error:(14, 46) java: package sun.reflect.generics.reflectiveObjects does not exist